packages/example-next-13-pages-and-app/messages/de.json

```diff
@@ -0,0 +1,44 @@
+{
+  "Meta": {
+    "title": "next-intl Beispiel",
+    "description": "Das ist ein Beispiel, wie next-intl im `app`-Verzeichnis verwendet werden kann."
+  },
+  "Index": {
+    "title": "Start",
+    "description": "Das ist die Startseite.",
+    "rich": "Das ist <important>formatierter</important> Test.",
+    "globalDefaults": "<highlight>{globalString}</highlight>"
+  },
+  "Client": {
+    "title": "Client",
+    "description": "Dise Seite wird auf der Client-Seite initialisiert."
+  },
+  "Nested": {
+    "title": "Verschachtelt",
+    "description": "Das ist eine verschachtelte Seite."
+  },
+  "NextPage": {
+    "title": "Next 12 Seite",
+    "description": "Diese Seite wird über den pages Ordner aufgerufen."
+  },
+  "Navigation": {
+    "home": "Start",
+    "client": "Client-Seite",
+    "nested": "Verschachtelte Seite",
+    "next-page": "Next 12 Seite"
+  },
+  "NotFound": {
+    "title": "Diese Seite wurde nicht gefunden (404)"
+  },
+  "LocaleSwitcher": {
+    "switchLocale": "Zu {locale, select, de {Deutsch} en {Englisch} other {Unbekannt}} wechseln"
+  },
+  "Counter": {
+    "count": "Aktuell:",
+    "increment": "Erhöhen"
+  },
+  "ClientCounter": {
+    "count": "Aktuell: {count}",
+    "increment": "Erhöhen"
+  }
```

Comment:
I think you've started off with the advanced example, right? Can we reduce this to the minimum use case of having one page in the `app` folder and one in the `pages` folder to avoid duplicating too much code?